tensorbay/opendataset/AnimalPose/loader.py

```diff
@@ -42,9 +42,9 @@
 
 
 def AnimalPose5(path: str) -> Dataset:
-    """Dataloader of the `5 Categories Animal Pose`_ dataset.
+    """Dataloader of the `5 Categories Animal-Pose`_ dataset.
```

Comment:
the `stardard TensorBay2.3` should be explained in the commit message body.

Reply:
The `standard TensorBay2.3` should be explained in the `commit message body` not PR description.